ingest/rules/nextclade.smk

```diff
@@ -1,86 +1,99 @@
-
-rule nextclade_dataset:
-    output:
-        temp("mpxv.zip"),
-    shell:
-        """
-        nextclade2 dataset get --name MPXV --output-zip {output}
-        """
+import sys
 
 
-rule nextclade_dataset_hMPXV:
+rule get_nextclade_dataset:
     output:
-        temp("hmpxv.zip"),
+        temp("data/mpxv.zip"),
+    params:
+        dataset_name="MPXV",
     shell:
-        """
-        nextclade2 dataset get --name hMPXV --output-zip {output}
+        r"""
+        nextclade3 dataset get \
+            --name {params.dataset_name:q} \
+            --output-zip {output:q}
         """
 
 
-rule align:
+rule run_nextclade:
```

Comment:
log/benchmark?

Reply:
Added in https://github.com/nextstrain/mpox/pull/281/commits/42c10401366712a3894c067656b3b46b33c8a7fa